app/google-services.json

```diff
@@ -0,0 +1,40 @@
+{
+  "project_info": {
+    "project_number": "220408294534",
+    "firebase_url": "https://simplye-nypl.firebaseio.com",
+    "project_id": "simplye-nypl",
+    "storage_bucket": "simplye-nypl.appspot.com"
+  },
+  "client": [
+    {
+      "client_info": {
+        "mobilesdk_app_id": "1:220408294534:android:594dde26dd9d35c301f507",
+        "android_client_info": {
+          "package_name": "org.nypl.simplified.simplye"
+        }
+      },
+      "oauth_client": [
+        {
+          "client_id": "220408294534-4vee7v894mol1bmq3sfqasq56i8jfnnj.apps.googleusercontent.com",
+          "client_type": 3
+        }
+      ],
+      "api_key": [
+        {
+          "current_key": "AIzaSyBkM8WKKf-LCL0dMldtvKx5dj6eVAmpCew"
```

Comment:
Is it OK to check this into the repos?

Reply:
@io7m Good catch, we should probably take this file out. I'll throw an error if the file is not present in the project when attempting to build.

Reply:
@io7m I can put the google-services.json file in the NYPL credentials repository.

Reply:
Yep, that's where it should go if it's private.